lib/modules/apostrophe-utils/lib/api.js

```diff
@@ -136,15 +136,19 @@ module.exports = function(self, options) {
     var i;
     var css = '';
     var dash = false;
+    var start = true;
     for (i = 0; (i < camel.length); i++) {
       var c = camel.charAt(i);
       var lower = ((c >= 'a') && (c <= 'z'));
       var upper = ((c >= 'A') && (c <= 'Z'));
       var digit = ((c >= '0') && (c <= '9'));
       if (!(lower || upper || digit)) {
-        dash = true;
+        if (!start) {
```

Comment:
Same here with `i === 0`. Fewer things to keep track of.

Reply:
See above re: doing the right thing if there's more than one non-alphanumeric character at the start.